### ISSUE TITLE
Failed to build Golang dde packages with Golang 1.15

### ISSUE THREAD

**Issue by cheese** [linuxdeepin/go-lib#21]
Dde-api (and also dde-daemon and startdde) failed to build on Fedora 33 with Golang 1.15.

The main error is the same for all the three packages:

```
/usr/share/gocode/src/pkg.deepin.io/lib/asound/asound.go:1354:17: cannot use _Ctype_enum_snd_mixer_selem_regopt_abstract(val) (type _Ctype_enum_snd_mixer_selem_regopt_abstract) as type uint32 in assignment
```

go-lib tested with 5.5.0.0 and 5.0.0.
Golang version 1.15 rc1.

Full build log: https://download.copr.fedorainfracloud.org/results/cheeselee/deepin-f33-fix/fedora-rawhide-x86_64/01593753-deepin-api/build.log.gz

**Comment by electricface** [comment]
fix at branch uos

**Comment by xmuli** [comment]
Fix submission  [#130ac7fb](https://github.com/linuxdeepin/go-lib/commit/130ac7fb2040295440beb0f70a267d532f89fc05)
